site/resources/_gen/assets/scss/sass/main.scss_511aa33e99371f93fbf403479ebfd32e.content

```diff
@@ -0,0 +1 @@
font-weight:400;font-style:normal}.footer{padding:15px;text-align:center;background-color:#252627}.footerLink{color:#f8f8ff}.header{font-family:fira mono,monospace;padding:0 20px;height:60px;display:flex;justify-content:space-between;align-items:center;width:760px}.headerWrapper{overflow:auto;box-sizing:border-box;background-color:#252627;display:flex;justify-content:center;flex-shrink:0}.headerLinks ul{list-style:none;padding:0;margin:0}.headerLinks ul li{display:inline;margin:5px}.headerLinks ul li a{color:#f8f8ff;text-decoration:none}.terminal{color:#f8f8ff;text-decoration:none}.hamb{cursor:pointer;float:right;display:none}.hamb-line{background:#f8f8ff;display:block;height:2px;position:relative;width:24px}.hamb-line::before,.hamb-line::after{background:#f8f8ff;content:"";display:block;height:100%;position:absolute;transition:all .2s ease-out;width:100%}.hamb-line::before{top:5px}.hamb-line::after{top:-5px}.side-menu{display:none}.side-menu:checked~.headerLinks{max-height:100%}.side-menu:checked~.hamb .hamb-line{background:0 0}.side-menu:checked~.hamb .hamb-line::before{transform:rotate(-45deg);top:0}.side-menu:checked~.hamb .hamb-line::after{transform:rotate(45deg);top:0}@media screen and (max-width:768px){.headerLinks{width:100%;height:100%;position:fixed;top:60px;left:0;background-color:#292a2d;overflow:hidden;max-height:0;transition:max-height .5s ease-out;display:flex;align-items:center;justify-content:center}.headerLinks ul{display:flex;flex-direction:column;align-items:center;justify-content:center}.hamb{display:block}}.postDate{float:right;color:#f8f8ff}.postTitle{color:#36c;text-decoration:none;text-transform:capitalize;font-family:fira mono,monospace;font-size:1.2em;float:left}.postHeader{overflow:hidden}.postsList{margin-top:20px}.postListItem{padding:20px;background-color:#2f2f2f;margin-bottom:20px}.postExcerpt{text-align:initial;text-decoration:none;color:#f8f8ff}.headerContainer{display:flex;justify-content:space-between}.noDecoration{text-decoration:none}a,a:link,a:visited,a:hover,a:active{text-decoration:none}.socialNavbar ul{list-style:none;padding:0}.socialNavbar ul li{display:inline}.socialNavbar a{color:#f8f8ff;text-shadow:#2f2f2f 0 0 2px;-webkit-font-smoothing:antialiased;display:inline-block;margin:10px}.postWrapper{text-align:left}.postMetadata{font-family:fira mono,monospace;display:block;background:#252627;padding:10px}dt,dd{line-height:1.5}dl{display:grid;grid-template-columns:max-content auto}dt{grid-column-start:1}dd{grid-column-start:2}dt::after{content:":"}.listHeader,.listContent{text-align:left}.indexWrapper{display:flex;flex-direction:column;justify-content:center}.indexHeader{font-size:3em}.content{text-align:center;margin:0 auto;flex:1 auto;padding:20px}@media only screen and (min-width:992px){.content{display:flex;width:760px}}.vertical{display:flex;flex-direction:column;justify-content:center}.flexWrapper{display:flex;flex-direction:column;justify-content:center;min-height:100vh}.main{width:100%;height:100%}
```

Comment:
Those are cache files and are not generated by default from hugo. Those were present due to submodule of nightfall. Since we  have a shift of the module it now caches the new module. 
https://gohugo.io/getting-started/directory-structure/#:~:text=resources,created%20by%20default.

Reply:
That's good enough for me :)